docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -396,8 +395,8 @@ They are questions present to the user, who can answer the question with either
 NOTE: Every procedure in a task or subtasks requires a verification.
 
 .Prerequisites
-* You have created a repo using the {walkthrough-name} template.
-* You have added the {walkthrough-name} repo to your cluster as described in the {publishing-walkthrough} {walkthrough-name}.
+* You have created a repository using the {walkthrough-name} template.
+* You have added the {walkthrough-name} repository to your cluster as described in the {publishing-walkthroughs} {walkthrough-name}.
```

Comment:
outta curiousity, is {publishing-walkthrough} currently broken?

Reply:
{publishing-walkthrough} does not exist but {publishing-walkthroughs} (note the 's') does.  {publishing-walkthrough} was not working because of this.  I am assuming that it was a typo and changed {publishing-walkthrough} to {publishing-walkthroughs} in the text.